### PR TITLE
Gluon Maps sample

### PR DESCRIPTION
**Pull request by abhinayagarwal**
This PR adds another sample to the repository which uses Gluon Maps

### CODE REVIEW

File: Maven/HelloMaps/pom.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-controls</artifactId>
+            <version>${javafx.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.gluonhq</groupId>
+            <artifactId>maps</artifactId>
+            <version>2.0.0-ea+2-SNAPSHOT</version>
```
Comment (jperedadnr) [inline Maven/HelloMaps/pom.xml]: I take there will be a release of Maps before the sample can be published?

Comment (abhinayagarwal) [reply Maven/HelloMaps/pom.xml]: This sample uses SNAPSHOT, so it will work. If we wish to not use `*-SNAPSHOT` then we do need a release.

**Comment by abhinayagarwal** [comment]
Can this be reviewed?

File: Maven/HelloMaps/pom.xml (hunk 1 above)
Comment (jperedadnr) [reply Maven/HelloMaps/pom.xml]: Can you release a new ea version of Maps?

Comment (tiainen) [reply Maven/HelloMaps/pom.xml]: Version `2.0.0-ea+3` has just been released.

**Comment by jperedadnr** [comment]
This sample should be added to https://github.com/gluonhq/gluon-samples

**Pull request closed without merging**